File: cakephp/vristo-cakephp-starter/src/Controller/TablesController.php
<?php
declare(strict_types=1);

namespace App\Controller;

class TablesController extends AppController
{
    public function tables()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/tables');
    }

    public function datatables_advanced()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/advanced');
    }

    public function datatables_alt_pagination()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/alt-pagination');
    }

    public function datatables_basic()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/basic');
    }

    public function datatables_checkbox()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/checkbox');
    }

    public function datatables_clone_header()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/clone-header');
    }

    public function datatables_column_chooser()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/column-chooser');
    }

    public function datatables_export()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/export');
    }

    public function datatables_multi_column()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/multi-column');
    }

    public function datatables_multiple_tables()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/multiple-tables');
    }

    public function datatables_order_sorting()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/order-sorting');
    }

    public function datatables_range_search()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/range-search');
    }

    public function datatables_skin()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/skin');
    }

    public function datatables_sticky_header()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/datatables/sticky-header');
    }
}

File: ruby/vristo-ruby-main/app/controllers/tables_controller.rb
class TablesController < ApplicationController
    def tables
        render layout: "default", :template => 'tables'
    end

    def datatables_advanced
        render layout: "default", :template => 'datatables/advanced'
    end

    def datatables_alt_pagination
        render layout: "default", :template => 'datatables/alt-pagination'
    end

    def datatables_basic
        render layout: "default", :template => 'datatables/basic'
    end

    def datatables_checkbox
        render layout: "default", :template => 'datatables/checkbox'
    end

    def datatables_clone_header
        render layout: "default", :template => 'datatables/clone-header'
    end

    def datatables_column_chooser
        render layout: "default", :template => 'datatables/column-chooser'
    end

    def datatables_export
        render layout: "default", :template => 'datatables/export'
    end

    def datatables_skin
        render layout: "default", :template => 'datatables/skin'
    end

    def datatables_multi_column
        render layout: "default", :template => 'datatables/multi-column'
    end

    def datatables_multiple_tables
        render layout: "default", :template => 'datatables/multiple-tables'
    end

    def datatables_order_sorting
        render layout: "default", :template => 'datatables/order-sorting'
    end

    def datatables_range_search
        render layout: "default", :template => 'datatables/range-search'
    end

    def datatables_sticky_header
        render layout: "default", :template => 'datatables/sticky-header'
    end
end
